Makefile

```diff
@@ -62,8 +55,10 @@ update-dependencies: ## Update go dependencies for all modules
 	# Update ./scripts/update go modules
 	cd ./scripts/update && go mod tidy && go mod verify
 
+$(GOLANGGCI_LINT): install
+
 check-linter: ## Install and run golang linter
-check-linter: | $(GOLANGGCI_LINT)
+check-linter: $(GOLANGGCI_LINT)
```

Comment:
Shouldn't this be `| $(GOLANGCI_LINT)`? We only care about if this file exists or not.